lib/default.nix

```diff
@@ -468,7 +492,7 @@ let
       # nix3 CLI output (`packages` output expects flat attrset)
       # FIXME: Find another way to make this work without introducing legacyPackages.
       #        May involve changing upstream home-manager.
-      legacyPackages = lib.optionalAttrs (homesNested != { }) standaloneHomeConfigurations;
+      legacyPackages = standaloneHomeConfigurations;
```

Comment:
iiuc, we only introduce this to have nested attributes because the home-manager cli doesn't understand flattened ones? If that's correct I think it'd be preferable to drop this here before it sees too much use and prioritize an upstream fix. What do you think?

(I know it's already in main, i just hadn't seen it before and took the change as there's already this PR touching it ;))